scripts/library_handler.py

```diff
@@ -54,7 +54,7 @@
 # some bad actors can't use the metadata correctly
 # and so need special treatment
 # -> see findLibAndVersion
-metaExceptions = ['pyside2', 'AMSC', 'PIL']
+metaExceptions = ['pyside2', 'AMSC', 'PIL', 'cvxpy']
```

Comment:
'cvxpy' is added via FARM plugin. It cannot be checked with the metadata. I followed the approach in the library_handler but this should be revised since it is not easily maintainable (overall for plugins).  @mandd @wangcj05 @joshua-cogliati-inl

Reply:
It seems this is due to an update of the library in conda-forge 2 days ago (see https://anaconda.org/conda-forge/cvxpy-base/files).

Reply:
> 'cvxpy' is added via FARM plugin. It cannot be checked with the metadata. I followed the approach in the library_handler but this should be revised since it is not easily maintainable (overall for plugins). @mandd @wangcj05 @joshua-cogliati-inl . I wonder why the regression system (for PR) did not catch this.

I changed the approach..if a library through a metadata is not found, we try the "slow approach".

Reply:
`skip_check` is another way to fix this. Example: `<hdf5 skip_check='True'/>`  I have no objection to fixing it in metaExceptions however.